cassandane/Cassandane/Mboxname.pm

```diff
@@ -132,15 +132,15 @@ sub _external_separator
     my ($self) = @_;
     die "No Config specified"
         unless defined $self->{config};
-    return $self->{config}->get_bool('unixhierarchysep', 'off') ? '/' : '.';
+    return $self->{config}->get_bool('unixhierarchysep', 'on') ? '/' : '.';
```

Comment:
Will this matter for any tests that maybe expected this to be off I wonder?

Reply:
Those tests will have configured it to be off, so it will be set to off in the config, so they will get back the false value they have set.  This line is fixing a bug where it assumed missing meant 'off', but actually missing means 'on'.

It matters, in that tests that assumed the wrong thing and were broken will now assume the right thing and work!  (Except that no such tests existed until I started writing these, and found the bug.)